addons/pos_sale/report/sale_report.py

```diff
@@ -97,6 +101,13 @@ def _query(self, with_clause='', fields={}, groupby='', from_clause=''):
             u.factor,
             pos.crm_team_id
         '''
-        current = '(SELECT %s FROM %s GROUP BY %s)' % (select_, from_, groupby_)
+        return groupby_
+
+    def _query(self, with_clause='', fields=None, groupby='', from_clause=''):
+        if not fields:
+            fields = {}
+        res = super(SaleReport, self)._query(with_clause, fields, groupby, from_clause)
```

Comment:
```suggestion
        res = super()._query(with_clause, fields, groupby, from_clause)
```
Specifying the class is not necessary ;)